src/components/Navbar.js

```diff
@@ -0,0 +1,16 @@
+const Navbar = () => (
+  <>
+    <nav className="navbar">
+      <div className="mainheading">
+        <h1> Math Magicians </h1>
+      </div>
+      <div className="links">
+        <a href="/">Home</a>
+        <a href="/Calculator">Calculator</a>
+        <a href="/Quote">Quote</a>
+      </div>
```

Comment:
## [ OPTIONAL CHANGE ]

- [ ] For a good visual and good user experience, it would be nice if you could make the page titles have white text color. Currently, their colors almost match the background color of the page and that is making them hard to be seen at a glance. They are the main links to accessing the different pages and hence, it will be good to make them stand out in white color. I trust you can adjust that 👍 
- [ ] Give the links the same white color as the Math Magicians logo title 👍

![image](https://user-images.githubusercontent.com/50941074/190209722-b2810b22-5f90-429e-b21b-c02d9aa28e97.png)

Reply:
Hi  @AmaduKamara,
  Thanks for the review. I have implemented the changes you asked thanks.